interpreter/cling/lib/Interpreter/Value.cpp

```diff
@@ -135,16 +135,54 @@ namespace {
 namespace cling {
 
   Value::Value(const Value& other):
-    m_Storage(other.m_Storage), m_StorageType(other.m_StorageType),
+    m_Storage(other.m_Storage), m_NeedsManagedAlloc(other.m_NeedsManagedAlloc),
+    m_TypeKind(other.m_TypeKind),
     m_Type(other.m_Type), m_Interpreter(other.m_Interpreter) {
     if (other.needsManagedAllocation())
       AllocatedValue::getFromPayload(m_Storage.m_Ptr)->Retain();
   }
 
+  static Value::TypeKind getCorrespondingTypeKind(clang::QualType QT,
```

Comment:
I don't remember seeing an argument why "Corresponding" is better than "Storage"?